azure-pipelines.yml

```diff
@@ -14,8 +14,10 @@ variables:
 
 steps:
 - script: |
+    set -x
```

Comment:
why this here?

Reply:
Recommendation from @rizzkhan1 as CI was falling over on this PR. It prints a trace of commands which seems to have fixed CI.

Reply:
To display the following commands in the CI logs when executing

Reply:
this set -x is optional

Reply:
weird...